### PR TITLE
Basic tests

### PR DESCRIPTION
**Pull request by cinekqw1**

### CODE REVIEW

File: test/playerTest.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    });
+
+    it('player name can be changed', () => {
+      // Assert that the width can be changed.
+      let playerName = 'Tom'
+      player.name = playerName;
+      player.name.should.equal(playerName);
+    });
+
+    it('only accepts string values of names', () => {
```
Comment (pparuzel) [inline test/playerTest.js]: No need for testing that. We need to test game functionality overall not the library and small bugs

**Comment by pparuzel** [comment]
**TL;DR** Please fix code at line 36 (`test/playerTest.js`) and then we can merge

I added two commits.
The first removes getter and setter from player.js. We shouldn't be adding such changes in a commit that adds tests.
The second removes selenium log file which is unnecessary to track.

Due to my former change (`test/playerTest.js`), code at line 36 and more will not work I guess so that needs fixing.
Please, focus on smaller commits that provide more information about the changes.

All in all, we'll need more regression tests rather than checking if player's name is correct. That is validation which we haven't implemented yet.

**Review by pparuzel — APPROVED** [review]
Resolved #33 
Ready to merge
